Flat/src/PandaAnalyzer.cc

```diff
@@ -304,25 +317,33 @@ void PandaAnalyzer::SetDataDir(const char *s)
   OpenCorrection(cPUUp,dirPath+"moriond17/puWeights_80x_37ifb.root","puWeightsUp",1);
   OpenCorrection(cPUDown,dirPath+"moriond17/puWeights_80x_37ifb.root","puWeightsDown",1);
 
-  // electrons
-  OpenCorrection(cEleVeto,dirPath+"moriond17/scaleFactor_electron_summer16.root",
-                 "scaleFactor_electron_vetoid_RooCMSShape_pu_0_100",2);
-  OpenCorrection(cEleTight,dirPath+"moriond17/scaleFactor_electron_summer16.root",
-                 "scaleFactor_electron_tightid_RooCMSShape_pu_0_100",2);
-  OpenCorrection(cEleReco,dirPath+"moriond17/scaleFactor_electron_reco_summer16.root",
-                 "scaleFactor_electron_reco_RooCMSShape_pu_0_100",2);
-
-  // muons
-  OpenCorrection(cMuLooseID,dirPath+"moriond17/muon_scalefactors_37ifb.root",
-                 "scalefactors_MuonLooseId_Muon",2);
-  OpenCorrection(cMuLooseIso,dirPath+"moriond17/muon_scalefactors_37ifb.root",
-                 "scalefactors_Iso_MuonLooseId",2);
-  OpenCorrection(cMuTightID,dirPath+"moriond17/muon_scalefactors_37ifb.root",
-                 "scalefactors_TightId_Muon",2);
-  OpenCorrection(cMuTightIso,dirPath+"moriond17/muon_scalefactors_37ifb.root",
-                 "scalefactors_Iso_MuonTightId",2);
-  OpenCorrection(cMuReco,dirPath+"moriond17/Tracking_12p9.root","htrack2",1);
-
+  if (analysis->complicatedLeptons) {
+   TString dirPath1 = TString(gSystem->Getenv("CMSSW_BASE")) + "/src/";
```

Comment:
Few things:

- let's not use `TSystem::GetEnv` in here as it can break things depending on the running environment
- let's use a more descriptive name than `dirPath1` 
- input data should go in `PandaAnalysis/data`. That's what that directory is meant for. I don't want an external dependency on `MitAnalysisRunII`. The *relevant* files get checked into `PandaAnalysis/data` (please no `git add *`)

Reply:
will move the dependencies and provide links in comments to the original file